react/package.json

```
@@ -11,15 +11,20 @@
     "author": "hey@posthog.com",
     "license": "MIT",
     "scripts": {
-        "clean": "rimraf -r dist",
         "prebuild": "npm run clean",
-        "build": "tsc",
         "start": "tsc-watch",
-        "test": "jest"
+        "clean": "rimraf -r dist",
+        "build": "yarn clean && cross-env NODE_ENV=production rollup -c rollup.config.ts",
+        "test": "cross-env BABEL_ENV=test jest --runInBand",
```

I understand that you may have added `--runInBand` to help debug issues with running the tests. I suggest adding a separate command to avoid slowing down tests during build:
```suggestion
        "test": "jest",
        "test:debug": "jest --runInBand",
```